### PR TITLE
rough start to state quick start guide API section

### PR DESCRIPTION
**Pull request by echappen**
Addresses #483

### CODE REVIEW

**Comment by echappen** [comment]
@ryanhofdotgov I figured this would remain WIP until we finalize more elements of the API, but wanted to externalize the points we talked about the other day:
- the NAC API is a single entity (even though they touch different subsystems)
- This top-level guide would reference the more detailed documentation for each subsystem as the Single Source of Truth
- Authentication to all endpoints is token-based, any dev would need to contact us for access

Let me know if this is going in the right direction.

**Comment by ryanhofdotgov** [comment]
@echappen Definitely the right direction 👍 

I'm anticipating there will be some of those workflow diagrams from @juliaklindpaintner's deck on this page, or perhaps put them on the main README and just reference them here.

**Comment by juliaklindpaintner** [comment]
@echappen I edited the diagrams to correspond more directly to the process described here — they're the first few slides in [this deck](https://docs.google.com/presentation/d/1MwRuw6EcLkcpHuEwGXgZ9_1RRAFO7O9SeCdCsRbZA6k/edit#slide=id.gc9a83cae3a_0_101). I didn't have one that represented the lookup workflow, so I added that. We can work together to refine them and then save them out for this or any other page in the repo. Feel free to leave comments if you see things that should be different!

**Comment by echappen** [comment]
@juliaklindpaintner Thanks! One thing to note:

When starting to write API specs for for Match ID Lookup, the engineering team realized the name "Match ID" was confusing us in a number of ways. So we switched the name to "Lookup ID" in the API spec to differentiate it more from the active matching process. If this is an issue with product/design, let's chat!

**Comment by juliaklindpaintner** [comment]
@echappen I don't see why that would be an issue! Would it make sense to rename the associated API endpoint accordingly?

File: docs/quick-start-guide-states.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+| production | actual data used in the production system | [coming soon] |
+
+### Endpoints Overview
+
+| endpoint | description | Type | Parameters | Response | Authentication |
+|---|---|---|---|---|---|
+| `/BulkUpload` | uploads bulk participant data to the system | POST | a CSV file [in this format](../etl/docs/bulk-import.md) | [coming soon] | Key-based, contact a developer for access |
+| [coming soon] | query the system for status on data processing from a bulk upload | POST | [coming soon] | [coming soon] | Key-based, contact a developer for access |
+| `/query` | query for matches | POST | refer to the [OpenApi Schema](../match/docs/openapi/orchestrator/index.yaml) | refer to the [OpenApi Schema](../match/docs/openapi/orchestrator/index.yaml) | Key-based, contact a developer for access |
+| `matchi_ids/:id` | Match ID Lookup | GET | match ID | [coming soon] | Key-based, contact a developer for access |
```
Comment (juliaklindpaintner) [inline docs/quick-start-guide-states.md]: Maybe this should be renamed Lookup ID here? It looks like this endpoint also may have a typo — an extra i after match?

Comment (echappen) [reply docs/quick-start-guide-states.md]: You may be looking at an outdated version of the file. I updated them [here](https://github.com/18F/piipan/blob/7b7c147c26c1a589edbc592b1300155a085f9a37/docs/quick-start-guide-states.md).

**Comment by juliaklindpaintner** [comment]
Whoops, sure was! Sorry about that!

__


Julia Lindpaintner (she/her)

18F <https://18f.gsa.gov/> | Technology Transformation Service
<http://www.gsa.gov/portal/category/25729>

General Services Administration (GSA)
<http://www.gsa.gov/portal/category/100000>

312.813.7727 | ***@***.***

Follow 18F on Twitter <https://twitter.com/18F>

File: docs/quick-start-guide-states.md (hunk 1 above)
Comment (juliaklindpaintner) [reply docs/quick-start-guide-states.md]: Whoops, sure was! I'm sorry about that!

**Comment by echappen** [comment]
@ryanhofdotgov I marked this as ready for review, I propose we merge and add docs as they're created

**Review by ryanhofdotgov — APPROVED** [review]
OK - sounds good. Before we send out to states, I think in a follow up issue(s) we'd want to:
* ditch the pre-prod/prod rows – we need to check with our partner before publishing here
* Consider how we present the match, lookup id, status APIs a single unified API. The fact they are 3 separate Function Apps is an implementation detail that shouldn't drive the documentation I think.
* Have a nice OpenAPI generated set of HTML documentation to link to.

**Comment by ryanhofdotgov** [comment]
Any other things we should specifically address in a follow up?

**Comment by echappen** [comment]
That list looks good to me. Added them to #623.

**Pull request merged**